### PR TITLE
Add more fields to SimpleAlbum, SimpleTrack from api

### PR DESCRIPTION
**Pull request by rociiu**
Screen Shot 2019-07-30 at 11 38 50 PM

### CODE REVIEW

File: track.go [hunk 1]
```diff
@@ -33,6 +35,8 @@ type SimpleTrack struct {
 	// DiscNumber.
 	TrackNumber int `json:"track_number"`
 	URI         URI `json:"uri"`
+	Popularity  int `json:"popularity"`
+        ExternalIDs map[string]string `json:"external_ids"`
```
Comment (zmb3) [inline track.go]: Looks like you need to run `gofmt`

**Comment by zmb3** [comment]
I think this change is incorrect.  I don't see any of these changes documented in the object model:
- https://developer.spotify.com/documentation/web-api/reference/object-model/#track-object-simplified
- https://developer.spotify.com/documentation/web-api/reference/object-model/#album-object-simplified

Here's the [interesting part](https://developer.spotify.com/documentation/web-api/reference/player/get-recently-played/): 
> On success, the HTTP status code in the response header is 200 OK and the response body contains an array of play history objects (wrapped in a cursor-based paging object) in JSON format. The play history items each contain the context the track was played from (e.g. playlist, album), the date and time the track was played, and a track object (simplified). On error, the header status code is an error code and the response body contains an error object. If private session is enabled the response will be a 204 NO CONTENT with an empty payload.

So I think what we need to do is add a new `PlayHistory` type and not touch `SimpleTrack` or `SimpleAlbum`

**Comment by rociiu** [comment]
@zmb3 i've corrected the format, also submitted a issue in https://github.com/spotify/web-api/issues/1314 .

**Comment by zmb3** [comment]
Thanks @rociiu - let's see what they say on the upstream issue.

**Comment by strideynet** [comment]
Closing as abandoned.

**Pull request closed without merging**